chat-sdk-ui/src/main/java/co/chatsdk/ui/threads/PublicThreadsFragment.java

```diff
@@ -30,6 +33,39 @@
 public class PublicThreadsFragment extends ThreadsFragment {
 
 
+    @Override
+    public void initViews() {
+        super.initViews();
+
+        Disposable d = adapter.onLongClickObservable().subscribe(thread -> {
```

Comment:
Can this be moved to the threads fragment base class? I expect the code is the same for both public and private threads?